modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/TransformSpec.scala

```diff
@@ -0,0 +1,32 @@
+package com.snowplowanalytics.snowplow.enrich.common.enrichments
+
+import cats.syntax.option._
+import com.snowplowanalytics.snowplow.enrich.common.adapters.RawEvent
+import com.snowplowanalytics.snowplow.enrich.common.loaders.CollectorPayload
+import com.snowplowanalytics.snowplow.enrich.common.outputs.EnrichedEvent
+import com.snowplowanalytics.snowplow.enrich.common.SpecHelpers.MapOps
+import org.specs2.mutable.Specification
```

Comment:
Usually we separate packages that come from different artifacts with newlines, and we finish by Snowplow ones